docs/src/classes.md

```diff
@@ -30,7 +31,10 @@ satisfied if the evaluation of this polynomial is an integer.
 
 
 ```@docs
-ortho2norm
-ortho2scalar
-classmult
+norm(t::CharTable, class::Int64)
+norm(t::SimpleCharTable{T}, class::Int64) where T <: NfPoly
+scalar_product(t::CharTable, class1::Int64, class2::Int64)
+scalar_product(t::SimpleCharTable{T}, class1::Int64, class2::Int64) where T <: NfPoly
+class_multiplication_coefficient(t::CharTable, class1::Int64, class2::Int64, class3::Int64)
+class_multiplication_coefficient(t::SimpleCharTable{T}, class1::Int64, class2::Int64, class3::Int64) where T <: NfPoly
```

Comment:
I *think* you can write this as
```suggestion
norm(t::Table, class::Int64)
scalar_product(t:: Table, class1::Int64, class2::Int64)
class_multiplication_coefficient(t:: Table, class1::Int64, class2::Int64, class3::Int64)
```

Reply:
(This is of course not important. From my POV you can merge this as-is as soon as the tests passed)